todo-mvc-kitchensink/src/containers/TodoDetailsContainer.ts

```diff
@@ -0,0 +1,14 @@
+import { Container } from '@dojo/widget-core/Container';
```

Comment:
Container for the todo details dialog that injects the todo object that is being edited